subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/report/Report.kt

```diff
@@ -231,31 +231,24 @@ interface Report : ReadReport {
         }
 
         override fun sendLostTests(lostTests: List<AndroidTest.Lost>) {
-            logger.critical(
```

Comment:
unrelated - this causes the build log to contain too many lines at the end
Now you will see this at the build-verdict